registry_pvc.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     - ReadWriteMany
   resources:
     requests:
-      storage: 100Gi
+      storage: 20Gi
```

Comment:
@anjannath After that if a user again request for 20G storage does it able to provision it or is it going to fail because 20GB already provided and only 10G remaining?

Reply:
have to check this again, i think anything with less then 30G should work

Reply:
verified it can provision new ones even though the available free space doesn't match up, looks like all the pvs the csi driver creates has capacity 30G which _i guess_ it takes from the size of the volume

```
[anjan@anath-test linux]$ ./oc get pvc
NAME            STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS                   AGE
task-pv-claim   Bound    pvc-ed92ac60-9ae9-445e-b5d4-1a15c22cc74d   30Gi       RWO            crc-csi-hostpath-provisioner   55s
[anjan@anath-test linux]$ ./oc get all -n testingns
NAME              READY   STATUS    RESTARTS   AGE
pod/task-pv-pod   1/1     Running   0          97s
[anjan@anath-test linux]$ ./oc get pvc -n openshift-image-registry
NAME                         STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS                   AGE
crc-image-registry-storage   Bound    pvc-517e0782-d71a-4bfe-8cfe-aba1941a6c19   30Gi       RWX            crc-csi-hostpath-provisioner   67m
```